docs/start/framework/react/comparison.md

```
@@ -35,6 +35,8 @@ Feature/Capability Key:
 | **Full-Stack Features**                                           | --                                               | --                                            | --                                         |
 | SSR                                                               | ✅                                               | ✅                                            | ✅                                         |
 | Streaming SSR                                                     | ✅                                               | ✅                                            | ✅                                         |
+| Partial Pre Rendering PPR
```

markdownlint-cli2 (0.23.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)